openfecwebapp/templates/partials/search-results-alternatives.html

```diff
@@ -1,17 +1,19 @@
 <div class="message message--alert">
   {% if not results %}
-  <h3>Sorry, there are no results for your search "{{ query }}"</h3>
+  <h2 class="message__title">No results</h2>
+  <p>Sorry, no results match your search, <span class="t-bold">“{{ query }}.”</span></p>
```

Comment:
Smart-like quotes here as well; again, is this intentional?  If it is, we should be using the HTML entities that correspond to them instead: https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references